frontend-ssr/pages/api/user/login.js

```diff
@@ -0,0 +1,38 @@
+//Imports
+const User = require("../../../models/User.schema");
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+const { loginValidation } = require("../../../utils/validation/userValidation")
+
+//Vars
+const error_message = { status: "fail", message: 'Email or Password is incorrect' }
+
+export default async (req, res) => {
+    let { method, body } = { ...req };
+    switch (method) {
+        case "POST":
+            //destruct the req
+            let { email, password } = { ...body };
```

Comment:
Same as above with the destructuring. Please change in all places where you're using like this.